runtime/compiler/runtime/JITServerIProfiler.cpp

```diff
@@ -163,7 +163,7 @@ JITServerIProfiler::profilingSample(uintptr_t pc, uintptr_t data, bool addIt, bo
    if (addIt)
       return NULL; // Server should not create any samples
 
-   TR_ASSERT(false, "not implemented for JITServer");
```

Comment:
I am not sure this should be converted - a) it is not enabled by default right now and seems a bit spurious and b) it isn't really unimplemented - it just doesn't do anything. @mpirvu do you know what the fate of this assert should be?

Reply:
This is more of a "we should never get here for JITServer". So the correct implementation is:
`TR_ASSERT_FATAL(false, "profilingSample(pc...) should not be called on JITServer");`

Reply:
Updated.